packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -193,9 +193,33 @@ export const contract = async (
   });
 
   const zoeTools = makeZoeTools(zcf, vowTools);
+  const { advanceFunds } = orchestrateAll(
+    // @ts-expect-error flow membrance type debt
+    { advanceFunds: flows.advanceFunds },
+    {
+      chainHub, // XXX excess authority?
+      feeConfig,
+      getNobleICA,
+      log: makeTracer('AdvanceFunds'),
```

Comment:
This kind appears in the baggage. @mhofman does a change in this object identity affect upgrade? (It will ofc change each time the contract starts again)

I wonder if the `Orchestrator` itself should provide a logging facility instead. We have some related issues about tracing through vstorage but not console logging,
- https://github.com/Agoric/agoric-sdk/issues/9066
- https://github.com/Agoric/agoric-sdk/issues/10421

Reply:
Yeah we should have logging better integrated, but I think we can punt on that.

No concerns with a log function. There is a kind defined, it just requires that we keep providing a `log` function in the future (the host power doesn't need to be the same function between upgrades, that's actually not possible and why we have to wrap these for now)